### DISCUSSION TITLE
Any repo out there that I can deploy to benchmark pocketbase?

### DISCUSSION THREAD

**Discussion by silenceofthewans** [pocketbase/pocketbase#1629] [Q&A] [answered]
I have a digital ocean trial and want to benchmark pocketbase at different machine sizes to see if this will fit my project. I've seen it mentioned here that pocketbase can handle thousands of concurrrent connection. How did you benchmark it is there a repo so I can test various hosts and machine configurations.

**Comment by ganigeorgiev** [comment, marked as answer]
>  How did you benchmark it is there a repo so I can test various hosts and machine configurations.

My early tests were with ab on Hetzner CX21 (2vCPUs, 4GB) mainly for GET queries with a basic filter and expand against several test collections (20k, 50k and 100k records). All tests were performed against the non-cgo driver and without extra indexes or optimizations (the CGO driver could be ~2x faster for some queries but it complicates cross compilation and I don't recommend it unless you really need it).

There is no official test bed and benchmarks repo yet. I'll try to prepare one once we reach a more stable version. Feel free to post your own repo if you find time and manage to create one.
